### DISCUSSION TITLE
Should we show tooltip for disable mood or disable button

### DISCUSSION THREAD

**Discussion by ashikjs** [primefaces/community#169] [PrimeNG] [answered]
[#13797](https://github.com/primefaces/primeng/issues/13797) this issue reported by AlexisDefranoux. But i am confusion here. What is valid case or what should be expected here.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Tooltip shouldn't be triggered when the button is disabled, this is not a bug. It's how it should be since the disabled button cannot receive any events. It's by design and it should stay as it is. 

I've closed the issue.
